core/config/genesis/prod.json

```diff
@@ -1 +1,62 @@
-{}
+{
+  "genesis_time": "2024-09-05T00:00:00.0000Z",
+  "chain_id": "audius-mainnet-test",
+  "initial_height": "0",
+  "consensus_params": {
+    "block": {
+      "max_bytes": "104857600",
+      "max_gas": "10000000"
+    },
+    "evidence": {
+      "max_age_num_blocks": "100000",
+      "max_age_duration": "172800000000000",
+      "max_bytes": "104857600"
+    },
+    "validator": {
+      "pub_key_types": [
+        "ed25519"
+      ]
+    },
+    "version": {
+      "app": "0"
+    },
+    "synchrony": {
+      "precision": "500000000",
+      "message_delay": "2000000000"
+    },
+    "feature": {
+      "vote_extensions_enable_height": "0",
+      "pbts_enable_height": "0"
+    }
+  },
+  "validators": [
+    {
+      "address": "35207ECB279B19AB53E0172F0E3AE47AC930D147",
```

Comment:
lowercase these addresses, comet won't match when the casing differs for some reason